gtfs_rt_validator_api.py

```diff
@@ -300,10 +306,12 @@ def validate_gcs_bucket_many(
                 project_id,
                 token,
                 verbose=verbose,
-                results_bucket=results_bucket + f"/{result_name_prefix}{idx}.parquet",
+                results_bucket=results_bucket
+                + f"/{result_name_prefix}_{row['calitp_itp_id']}_{row['calitp_url_number']}_{row['output_file_suffix']}.parquet",
```

Comment:
And do we want to start "better" partitioning now -- do we want to make this one of `date/type/itp id/url` or `date/itp id/url/type/` rather than just `_` delimited?

Reply:
That's fair, I think in this case since this is the output of a single job based on input parameters, I want to group it at the same level. Otherwise unless we decide to stop overwriting output, there's only one file per full subpath.

Reply:
from slack discussion, going to change to `date/itp/url` with one file per entity type; this will mirror what how the actual RT data is parsed out